docs/compatibility.md

```diff
@@ -3,6 +3,7 @@ This table serves as a guide, suggesting which versions of individual submodules
 
 | [ODE (this project)](https://github.com/usdot-jpo-ode/jpo-ode/releases) | [ACM](https://github.com/usdot-jpo-ode/asn1_codec/releases) | [PPM](https://github.com/usdot-jpo-ode/jpo-cvdp/releases) | [SEC](https://github.com/usdot-jpo-ode/jpo-security-svcs/releases) | [SDWD](https://github.com/usdot-jpo-ode/jpo-sdw-depositor/releases) | [S3D](https://github.com/usdot-jpo-ode/jpo-s3-deposit/releases) | [GJConverter](https://github.com/usdot-jpo-ode/jpo-geojsonconverter/releases) | [CMonitor](https://github.com/usdot-jpo-ode/jpo-conflictmonitor/releases) | [CVisualizer](https://github.com/usdot-jpo-ode/jpo-conflictvisualizer/releases) | [CVManager](https://github.com/usdot-jpo-ode/jpo-cvmanager/releases) |
 | ----------------- | --- | --- | --- | ---- | --- | ----------- | -------- | ----------- | ----------- |
+| 4.0.0 | 3.0.0 | 1.5.0 | 1.5.0 | 1.9.0 | 1.7.0 | 1.5.0 | 1.5.0 | 1.5.0 | 1.5.0 |
```

Comment:
```suggestion
| 4.0.0 | 3.0.0 | 1.5.0 | 1.5.0 | 1.9.0 | 1.7.0 | 2.0.0 | 1.5.0 | 1.5.0 | 1.5.0 |
```
The jpo-geojsonconverter is also incrementing major versions since it is not backwards compatible with 1.4.0.

Reply:
Addressed! Thanks for catching that. Should the major versions of the jpo-conflictmonitor and jpo-conflictvisualizer projects be incremented as well?

Reply:
The output of the jpo-geojsonconverter is the same as it was before so those two services shouldn't need major version increments.

Reply:
Sounds good, thanks Drew!